cs/ccxt/ws/Client.cs

```diff
@@ -461,11 +461,7 @@ public async Task Close()
             }
             foreach (var future in this.futures.Values)
             {
-                if (!future.task.IsCompleted)
```

Comment:
why remove the check from here?

Reply:
becausue moved into another file, look at it too

Reply:
however, this is the case when i dont think that I am doing it correctly, bcz i couldnt reproduce termittent TaskCompletionSource issues and I just "expect" these guards might be fix to avoid them, that's why this solely depends on what you think

Reply:
@ttodua yesterday I made a commit that tries to mitigate this issue, https://github.com/ccxt/ccxt/pull/23202
have you checked if after the commit we still get those errors?

Reply:
i haavent checked that.. ok lets see in the next few period, if issues will not pop up again, then good

Reply:
will turn this PR into draft, if smth, you can merge later.

Reply:
i've run several tests with this branch and i dont get that error. idk, you might also test out on your side

Reply:
the thing is I think this is more a race-condition problem, the state of the task might be changing in between the if (isCompleted) and the action inside the id that's why I added a lock in the other PR